doc/rst/src/getting_started.rst

```diff
@@ -1,71 +1,64 @@
   vectors. See :ref:`bfp_background` for an introduction to block floating-point. These functions
-  manage the exponents and headroom of input and output BFP vectors to avoid overflow and underflow 
+  manage the exponents and headroom of input and output BFP vectors to avoid overflow and underflow
   conditions.
 
-* :ref:`Vector/Array API <vect_api>` -- Lower-level API which is used heavily by the BFP API. 
+* :ref:`Vector/Array API <vect_api>` -- Lower-level API which is used heavily by the BFP API.
   As such, the operations available in this API are similar to those in the BFP API, but the user
   will have to manage exponents and headroom on their own. Many of these routines are implemented
   directly in optimized assembly to use the hardware as efficiently as possible.
 
-* :ref:`Scalar API <scalar_api>` -- Provides various operations on scalar objects. In particular, 
-  these operations focus on simple arithmetic operations applied to non-IEEE 754 floating-point 
+* :ref:`Scalar API <scalar_api>` -- Provides various operations on scalar objects. In particular,
+  these operations focus on simple arithmetic operations applied to non-IEEE 754 floating-point
   objects, as well as optimized operations which are applied to IEEE 754 ``floats``.
 
-* :ref:`Filtering API <filter_api>` -- Provides access to linear filtering operations, including 
+* :ref:`Filtering API <filter_api>` -- Provides access to linear filtering operations, including
   16- and 32-bit FIR filters and 32-bit biquad filters.
 
-* :ref:`Fast Fourier Transform (FFT) API <fft_api>` -- Provides both low-level and block 
+* :ref:`Fast Fourier Transform (FFT) API <fft_api>` -- Provides both low-level and block
   floating-point FFT implementations.  Optimized FFT implementations are provided for real signals,
   pairs of real signals, and for complex signals.
 
-* :ref:`Discrete Cosine Transform (DCT) API <dct_api>` -- Provides functions which implement the 
-  `type-II <https://en.wikipedia.org/wiki/Discrete_cosine_transform#DCT-II>`_ ('forward') and 
-  `type-III <https://en.wikipedia.org/wiki/Discrete_cosine_transform#DCT-III>`_ ('inverse') DCT for 
+* :ref:`Discrete Cosine Transform (DCT) API <dct_api>` -- Provides functions which implement the
+  `type-II <https://en.wikipedia.org/wiki/Discrete_cosine_transform#DCT-II>`_ ('forward') and
+  `type-III <https://en.wikipedia.org/wiki/Discrete_cosine_transform#DCT-III>`_ ('inverse') DCT for
   a variety of block lengths. Also provides a fast 8x8 two dimensional forward and inverse DCT.
 
 
 All APIs are accessed by including the single header file:
 
 .. code-block:: c
-  
-    #include "xmath/xmath.h"
-
 
-Building
---------
-
-This library makes use of the CMake build system. See the 
-`GitHub readme <https://github.com/xmos/lib_xcore_math>`_ for instructions on
-obtaining this library's source code and including it in your application's build.
+    #include "xmath/xmath.h"
 
 Usage
------
+======
```

Comment:
(nitpick) Underline is longer than necessary

Reply:
nitpicking fine!! :)